### PR TITLE
Add model stream schema validation.

### PR DESCRIPTION
**Pull request by pawartur**
## Description

Add ajv strict validation of json-schemas to Model

Use draft 2020-12 for validation: https://ajv.js.org/json-schema.html#draft-2020-12

## How Has This Been Tested?

- [x] Wrote tests for the new `SchemaValidation` class in `packages/stream-model-handler`
- [x] Wrote tests for the `ModelHandler` in `packages/stream-model-handler` to make sure commits are only applied to Models, if their schema conforms to JSON Schema standard (unless they don't have schema)

## PR checklist

Before submitting this PR, please make sure:

- [x] I have tagged the relevant reviewers and interested parties
- [ ] I have updated the READMEs of affected packages
- [ ] I have made corresponding changes to the documentation

## References:

Please list relevant documentation (e.g. tech specs, articles, related work etc.) relevant to this change, and note if the documentation has been updated.

### CODE REVIEW

File: packages/stream-model-handler/src/__tests__/model-handler.test.ts
Comment (stbrody) [inline packages/stream-model-handler/src/__tests__/model-handler.test.ts:322]:
```suggestion
  it('fails to apply signed commit with invalid schema', async () => {
```

File: packages/stream-model/src/model.ts
Comment (stbrody) [inline packages/stream-model/src/model.ts:21]: 🙏

File: packages/stream-model-handler/yarn.lock
Comment (stbrody) [inline packages/stream-model-handler/yarn.lock:1]: please remove this file.  Fine if you use yarn locally, but the "official" way to build ceramic is with npm so we don't want to check in files for other build systems into the repo

**Review by stbrody — COMMENTED** [review]
looks great, but please hold off on merging until after my PR is merged first, so that the two commits don't get squashed together into one

**Comment by stbrody** [comment]
@pawartur - base PR has been merged, can you please merge in `develop` and update this PR?

**Comment by pawartur** [comment]
> @pawartur - base PR has been merged, can you please merge in `develop` and update this PR?

Done, @stbrody , thanks :) !

File: packages/stream-model-handler/src/__tests__/model-handler.test.ts
Comment (stbrody) [reply packages/stream-model-handler/src/__tests__/model-handler.test.ts:322]: ping

**Review by stbrody — COMMENTED** [review]
lgtm, feel free to merge after committing my small change to the test description text

File: .gitignore
Comment (ukstv) [inline .gitignore:9]: Please, add these two to your local .gitignore.

Comment (ukstv) [inline .gitignore:9]: Please, add these two to your local .gitignore.

**Review by ukstv — APPROVED** [review]

**Pull request merged**
